megaavr/extras/LowPower.md

```diff
@@ -0,0 +1,635 @@
+# LowPower.cpp
+    This document is a comprehensive list of all the functions from LowPower.cpp. Each function includes descriptions for how they change the hardware to decrease the power draw for DxCore microcontrollers.
+    All information for the hardware changes and interrupts can be found in the AVR128DA48 datasheet from Microchip which can be found here: https://www.microchip.com/en-us/product/AVR128DA48
+
+## setSleepMode(SLPCTRL_SMODE_t sleepMode)
+Description: 
+
+    Sets the microcontroller's specified sleep mode, but does not enter sleep mode automatically.
+
+Arguments: 
+    
+    sleepMode - the type of sleep mode the microcontroller will use when it enters sleep mode.
```

Comment:
I would suggest using code tags for function arguments, names, and sample calls throughout. 

```suggestion
`sleepMode` - the type of sleep mode the microcontroller will use when it enters sleep mode.
```